### DISCUSSION TITLE
Is "Craft AI-driven interfaces effortlessly" the right positioning?

### DISCUSSION THREAD

**Discussion by yingpengsha** [ant-design/x#370] [Ideas]
I’ve been thinking about whether “AI-driven interfaces effortlessly” is the best way to position Ant Design X. It feels like this phrasing puts too much emphasis on “AI,” when in reality, this direction seems more like a continuation of the well-established concept of **Natural-language user interface (NLUI)** rather than something entirely new. Here are a few points I’d love to discuss with the community:

- **NLUI might align better with the core idea**
  NLUI is a more mature and accurate concept. At its heart, it’s about making communication between people and interfaces more natural. On the other hand, “AI-driven” seems to define interaction based on technology, but AI doesn’t actually invent new ways to interact—it simply enhances or rotates through existing trends like voice or text input.

- **AI won’t eliminate the need for interfaces**
  The purpose of user interfaces is to help people interact with machines more efficiently, and that need won’t disappear because of AI. AI might replace the people who design or code interfaces, but it won’t replace the interfaces themselves. The real game-changer would be something like brain-computer interfaces, which could fundamentally change how information is transmitted—not AI.
 
- **What should we focus on?**
  Instead of fixating on “AI,” we should think more deeply about what changes AI is bringing to how we exchange information. The key to great interface design isn’t following trends but understanding the core needs of human-machine interaction. If Ant Design X’s positioning could shift toward exploring more efficient ways for humans and machines to communicate, it could become a truly industry-defining tool—not just another library with an “AI” label.

These are just some of my thoughts—I’d love to hear yours!

**Comment by petercat-assistant** [comment]
Your thoughts on the positioning of Ant Design X are insightful and raise important considerations. Here are some reflections on the points you've made:

1. **NLUI vs. AI-driven**: The emphasis on "AI-driven" might indeed overshadow the more fundamental concept of Natural-language user interfaces (NLUI). NLUI focuses on making interactions more intuitive and natural, which aligns well with the core idea of enhancing user experience. AI, while a powerful tool, is more about augmenting existing interaction methods rather than creating entirely new paradigms.

2. **The Role of Interfaces**: You're correct in noting that AI won't eliminate the need for interfaces. Interfaces serve as the bridge between humans and machines, and while AI can enhance this interaction, it doesn't replace the necessity for well-designed interfaces. The future might hold more revolutionary changes like brain-computer interfaces, but for now, the focus should remain on improving existing methods.

3. **Focus on Communication**: Shifting the focus from merely being "AI-driven" to exploring how AI can improve communication between humans and machines is a valuable perspective. Understanding the core needs of human-machine interaction and leveraging AI to meet those needs can indeed position Ant Design X as a leader in the industry.

Your perspective encourages a more nuanced approach to positioning, one that considers the broader implications of AI and NLUI in interface design. Engaging with the community on these points could lead to a more refined and impactful positioning strategy for Ant Design X.

If you have more questions or need further assistance, feel free to ask!

<details>
<summary>🪧 Tips</summary>
For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.
</details>

**Comment by YumoImer** [comment]
首先，非常感谢你能够分享你对 `X` 的见解！

关于 `X` 这个名字的寓意：在 AGI 时代，用户界面的无限可能性。

我非常认同在当前阶段 LUI 确实是一个非常合适的交互方式。

不过未来充满了未知与变化，随着技术的进步，大家都在不断探索和发现新的可能性。

在当下 `X` 提供了一些 UIKit 以及 Tools 帮助开发者更好的搭建 LUI 应用，同时也会保持开放的心态，和大家一起边走边探索！

下图我感觉挺符合这个话题，所以我把它贴了过来，方便给大家进一步的讨论提供一些资料。

> 下图来自于我们设计师团队提出的 [RICH 设计范简介](https://x.ant.design/docs/spec/introduce-cn)
![](https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*cXCbRJO2Rl4AAAAAAAAAAAAADgCCAQ/fmt.webp)
